gw_challenge/ingested_program/ingestion.py

```diff
@@ -65,16 +69,31 @@ def print_pretty(text):
 
 def install_from_whitelist(req_file):
 
-    whitelist_file = os.path.join(input_dir, "whitelist.txt")
-    whitelist = open(whitelist_file, 'r').readlines()
+    whitelist = open("whitelist.txt", 'r').readlines()
```

Comment:
@egrace479 what's the reason for changing the path here? The `whitelist.txt` should be supplied either with inputs (as i put originally, eg in `/app/input_data/`) or we can add it with ingestion script, so here` /app/program/`. Or do you now provide the file in some other way? thanks

Reply:
Ah sorry, I was testing just the function and copied it over with the adjusted file read. I think we may want to have the whitelist embedded in the function or at least included in the ingestion program folder instead of with the input data (would make it much more complicated if we had to update it do to the size of our input data).